jgrapht-core/src/main/java/org/jgrapht/alg/cycle/WeakChordalityInspector.java

```diff
@@ -0,0 +1,443 @@
+/*
+ * (C) Copyright 2018, by Timofey Chudakov and Contributors.
```

Comment:
Before I forget: please keep the copyright header format identical. E.g. your first line should read:
 * (C) Copyright 2018-2018, by Timofey Chudakov and Contributors.

Headers are updated automatically; Deviation of the standard header messes this up.